core-bundle/src/Resources/contao/themes/flexible/basic.css

```diff
@@ -272,6 +271,7 @@ fieldset.checkbox_container,fieldset.radio_container {
 	cursor:not-allowed;
 }
 .tl_textarea {
+	height:336px;
```

Comment:
In case you wonder: TinyMCE 5 uses the textarea height for its overall height (menu bars + editable area) instead of just for the editable area, therefore I have increased the textarea height so the editable area is 240px high again.

Reply:
but that would apply to non-tinyMCE textareas as well, right?

Reply:
Yes, however, those textareas are resized automatically based on the content. I can still check if this can be optimized.

Reply:
Optimized in 62bd8adb33dbf31a96254f13c8d19364849aa222.